### ISSUE TITLE
Document why packer needed here

### ISSUE THREAD

**Issue by Arlington1985** [linkedin/oncall#253]
It's completely unknown why we use packer with docker installation? The installation finishes successfully even packer part fails/ Please document the purpose of it with just a few words at least

**Comment by houqp** [comment]
packer is only needed for building the docker image from source. if you are just consuming the docker image, then it's not required

**Comment by Arlington1985** [comment]
Yes but this part should be properly documented I think

**Comment by houqp** [comment]
I agree, feel free to send a PR to improve the doc. Or even better, create a raw Dockerfile and move away from packer. I think the current setup is an overkill.
